skycultures/almagest/description.en.utf8

```diff
@@ -1,34 +1,41 @@
-astronomers including mathematical and astronomical results. Pieced
+is based on the Almagest, a compilation of astronomical knowledge of former
+astronomers including results from observations and mathematical astronomy. Pieced
 together by Claudius Ptolemy, using older, as well as his own,
 observations, it was written in Greek, published in the Second
 Century and remained the main astronomical work for the following
 1500 years.  It was originally written with the title <i>Mathematike
 Syntaxis</i>, in 
 Ancient Greek.  The name <i>Almagest</i>,
 meaning &ldquo;The Largest [Book]&rdquo;, was developed from Arabian
-translations.  To create this sky culture the translation by Toomer
-(1984) was used. </p>
+translations.  To create this sky culture we used the English translation by Toomer
+(1984). </p>
 <p >
  The star catalogue consists of
-13 sections, called books.  In the seventh book, Ptolemy listed total
-1022 stars in 48 constellations (among which are about 5 star-like
+13 sections, called books.  In the seventh and eighth book, Ptolemy listed a total
+of 1022 stars in 48 constellations (among which are about 5 star-like
 objects like globular star clusters).  It is a table of coordinates
 of the stars with a description of their position in the
-constellation.  He used ecliptic coordinates.  As one of the first of
+constellation (such as "star at the ellbow of X, longitude, latitude, magnitude").  
+With regard to precession, Ptolemy used ecliptic coordinates.  As one of the first of
 his time, Ptolemy included the magnitude, a quantitative scale of
 stellar brightness, probably adopted from Hipparchus (which is not
 certain because Hipparchus&rsquo; catalog is not preserved).  The
 scale starts with the brightest star with the magnitude of 1, going
-down to magnitude of 6, for the faintest stars visible.  Based on the
-geocentric model the <i>Almagest</i> 
-presents the final displacement of the heliocentric model by
-Aristarch and mostly relies on mathematical calculation than on
-physical observations, which explains discrepancies in the
-coordinates written in the  <i>Almagest</i>. </p>
+down to magnitude of 6, for the faintest stars visible.
+</p>
+
+<p>For the computation of planet positions Ptolemy used a
+geocentric model (which is allowed because the observer always stands on Earth) and, thus, 
+contributed to the scientific believe that the heliocentric model by
+Aristarch was not needed. However, for the star catalogue of the <i>Almagest</i> 
+this does not matter at all because a star catalogue always represents a sort of inventory 
+of the fixed dots in the sky and Ptolemy's proclaimed aim of the catalogue is to provide a list
+for making a celestial globe (see Alm., VIII, 3). He uses ecliptical coordinates
+because being aware of precession he wants to ease the business of future globemakers as they 
+might wish to transform his catalogue into coordinates of their epoch. </p>
 
```

Comment:
Maybe add (for the non-initiated), or combine the previous sentence with:

Precession as known to Ptolemy shifts all stars parallel to the ecliptic, therefore updates to the epoch of the respective globe maker  could be applied by a simple subtraction of the value for ecliptic longitude.

Reply:
ok, please do so